src/Spectre.libGenetic.Tests/CrossoverOperatorTest.cpp

```diff
@@ -41,25 +49,25 @@ class CrossoverOperatorTest: public ::testing::Test
     const Seed SEED = 0;
     const Individual true_individual = Individual({ true, true, true, true });
     const Individual false_individual = Individual({ false, false, false, false });
-    CrossoverOperator crossover;
+    std::unique_ptr<CrossoverOperator> crossover;
```

Comment:
What is the reason behind this change?

Reply:
Just it was easier for me to create pointer in setUp function, and to be more consistent with other tests

Reply:
During the tests we need to recreate this object. It's much easier to go for `unique_ptr`.